app/code/community/TIG/MyParcel2014/Model/Api/MyParcel.php

```diff
@@ -805,6 +807,31 @@ protected function _getConsignmentData(TIG_MyParcel2014_Model_Shipment $myParcel
             $data['physical_properties']['weight'] = (int)$totalWeight;
         }
 
+        if($data['options']['package_type'] == self::DIGITAL_STAMP){
```

Comment:
Als een method te groot is, moet het opgedeeld worden. Je zou dus hier een eigen method van moeten maken

Reply:
Done!